CMakeLists.txt

```diff
@@ -5,20 +5,20 @@
 # IMPORTANT: Changes which affect binary results may not be quietly gated
 #            by CMake version.
 #
-# Debian 10 Buster, https://wiki.debian.org/LTS, EOL 2024:
-#  - CMake 3.13.4, https://packages.debian.org/buster/cmake
+# Debian 10 Buster, https://wiki.debian.org/LTS, EOL in June 2024:
+#  - CMake 3.18.4, https://packages.debian.org/buster-backports/cmake
 #
-# Ubuntu 22.04 Jammy, https://wiki.ubuntu.com/Releases, EOL 2032:
-#  - CMake 3.22.1, https://packages.ubuntu.com/jammy/cmake
+# Ubuntu 20.04 LTS Jammy, https://wiki.ubuntu.com/Releases, EOSS in April 2025:
+#  - CMake 3.16.3, https://packages.ubuntu.com/focal/cmake
 #
-# Visual Studio 17 2022, https://visualstudio.microsoft.com:
-#  - CMake 3.24
+# Ubuntu 22.04 LTS Jammy, https://wiki.ubuntu.com/Releases, EOSS in June 2027:
```

Comment:
Why are we listing multiple version of Ubuntu here. Seems unecessary to repeat things that are trivial to look up, and the only one that matters is our minimum supported?

Reply:
Thanks! Reworked.